src/apps/viewer/app.ts

```diff
@@ -423,6 +424,34 @@ export class Viewer {
         });
     }
 
+    async loadFullResolutionEMDBMap(emdbId: string, options: { isoValue: Volume.IsoValue, color?: Color }) {
```

Comment:
```suggestion
    loadFullResolutionEMDBMap(emdbId: string, options: { isoValue: Volume.IsoValue, color?: Color }) {
```

No need for the async here since you are no awaiting anything.